src/components/ButtonPanel.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Button from './Button';
+
+const ButtonPanel = () => (
+  <div>
+    <div className="board-row">
+      <Button name="AC" />
+      <Button name="+/-" />
+      <Button name="%" />
+      <Button name="÷" />
+    </div>
+    <div className="board-row">
+      <Button name="7" />
+      <Button name="8" />
+      <Button name="9" />
+      <Button name="X" />
+    </div>
+    <div className="board-row">
+      <Button name="4" />
+      <Button name="5" />
+      <Button name="6" />
+      <Button name="-" />
+    </div>
+    <div className="board-row">
+      <Button name="1" />
+      <Button name="2" />
+      <Button name="3" />
+      <Button name="+" />
+    </div>
+    <div className="board-row">
+      <Button name="0" />
+      <Button name="." />
+      <Button name="=" />
+    </div>
```

Comment:
### [Optional]
I think we could make use of two loops to make the Button instead of repeating ourselves. let's practice [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) code.

To do this; create an array of symbols and loop through the array using [map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.